core/iwasm/aot/aot_loader.c

```diff
@@ -123,6 +123,28 @@ GET_U64_FROM_ADDR(uint32 *addr)
     return u.val;
 }
 
+#if defined(BUILD_TARGET_XTENSA)
```

Comment:
using VRAM has two preconditions, one is xtensa target, another is xip mode. xip is a runtime variant read from file which need 4 bytes read
then we make VRAM enable/disable as compliling time not runtime? 
Or, we read xip mode using 4 bytes align anyway, and after that change the reading mode according to the XIP status?

Reply:
> using VRAM has two preconditions, one is xtensa target, another is xip mode. xip is a runtime variant read from file which need 4 bytes read then we make VRAM enable/disable as compliling time not runtime? 

Yes, had better make VRAM enable/disable as compiling time behavior, since for xtensa, not all situations require VRAM, it is only used in special situation. I believe there should be VRAM related macro defined, if not, you can define it in your system.